components/triggercmd/actions/trigger-command/trigger-command.mjs

```diff
@@ -0,0 +1,48 @@
+import triggercmd from "../../triggercmd.app.mjs";
+import axios from "axios";
+
+export default {
+  name: "Trigger a command",
+  version: "0.0.12",
+  key: "trigger-command",
+  description: "Runs a command on a computer.  Refer to the [TRIGGERcmd Forum](https://triggercmd.com/forum) to learn more.",
+  props: {
+    token: {
+      type: "string",
+      label: "Token from the Instructions page in your account at TRIGGERcmd.com",
+      secret: true
+    },
+    computer: {
+      type: "string",
+      label: "Computer Name",
+    },
+    trigger: {
+      type: "string",
+      label: "Trigger Name",
+    },
+    params: {
+      type: "string",
+      label: "Command parameters",
+      optional: true
+    }
+  },
+  type: "action",
+  methods: {},
+  async run() {
+    // Make an HTTP POST request using axios
+    const resp = await axios({
+      method: "POST",
+      url: `https://www.triggercmd.com/api/run/triggerSave`,
+      data: {
+        token: this.token,
+        computer: this.computer,
+        trigger: this.trigger,
+        params: this.params
+      },
+    });
+
+    // Retrieve just the data from the response
+    const { data } = resp;
+    console.log(data)
```

Comment:
Maybe you want to return the data, instead of just `console` it? Also, if you use `axios` from pipedream, you don't need to use `resp.data`. The response will automatically be the `data`

Reply:
Done